app/scenes/ChangeStudy/index.js

```diff
@@ -9,40 +9,44 @@ import {
   Left,
   Right,
   Icon,
-  Body,
   Text,
 } from 'native-base';
 import { Actions } from 'react-native-router-flux';
 import QRCodeScanner from 'react-native-qrcode-scanner';
 import styles from './styles';
+import { getSkin } from '../../services/network';
```

Comment:
`getSkin` is undefined

Reply:
Good catch, I forgot to add that change to the commit. Should be there now.